GTScript-Parallel-model.md

```diff
@@ -296,3 +296,118 @@ with computation(...):
         if field:
             a = a[0, 1, 0] # self assignment with offset (i.e. a read with offset and write)
 ```
+
+## Off-center writes in `K`
+
+The vertical direction, `K`, is special in weather and clinate applicaitons. It is used nut just for stencil computations but also for implicit skemes and other algorthms in physical parametrization.
+This is the reason why the vertical look is kept separate from tne horizontal IJ look in the parallel model.
+
+The vertical loop can have _directions_, indicated by the iteraiton policies FORWARD, BACKWARD and PARALLEL. To implement certain skemes it is natural to allow for off-center writes in the vertical direction, since the programmer can know in which order the values are accessed.
+
+<details>
+### Simple example
+We introduce the behavior and policies in GTScript by looking at one sample code executed the in three different iterations policies.
+
+```python
+with computation(FORWARD):  # Forward computation
+    with interval(start, end):
+        a[0,0,-1] = c
+        b[0,0,1] = 2 * a
+
+with computation(BACKWARD):  # Forward computation
+    with interval(end, start):
+        a[0,0,-1] = c
+        b[0,0,1] = 2 * a
+
+with computation(PARALLEL):  # Forward computation
+    with interval(start, end):
+        a[0,0,-1] = c
```

Comment:
I'd not translate this to pseudo-code but add a comment that it's ambiguous. In the future, I'd require GT4Py to emit an error.

Reply:
I wanted to show that a translation would not make sense. Probably the whole pseudocode here is unnecessary